pkg/apis/maistra/conversion/jaeger_test.go

```diff
@@ -39,7 +39,7 @@ func jaegerTestCasesV2(version versions.Version) []conversionTestCase {
 					"meshExpansion": globalMeshExpansionDefaults,
 					"enableTracing": false,
 					"proxy": map[string]interface{}{
-						"tracer": "zipkin",
+						"tracer": "none",
```

Comment:
Is this "none" value case sensitive ? From UI, I see tracing provider type value option is "None"

Reply:
nvm, I see the value lower case is correct one from: https://github.com/istio/istio/issues/45890

Reply:
https://github.com/maistra/istio/blob/maistra-2.5/operator/pkg/apis/istio/v1alpha1/values_types.pb.go#L118

> From UI, I see tracing provider type value option is "None"

This is only our value. Istio uses lowercase.